CHANGELOG.md

```diff
@@ -8,6 +8,52 @@
 % # " - [Bug Fix]"
 % # " - Fix the bug."
 
+## Version 0.9.0 (June 17th, 2021)
+
+### Summary
+
+The main addition introduced in this release is the implementation of a new error mitigation technique: (variable-noise) Clifford data regression ([arXiv:2005.10189](https://arxiv.org/abs/2005.10189), [arXiv:2011.01157](https://arxiv.org/abs/2011.01157)). This is structured as
+a Mitiq module called `mitiq.cdr`.
+
+Another important change is the integration with Amazon Braket, such that Mitiq is now compatible with circuits of type `braket.circuits.Circuit`. Moreover all the existing Mitiq integrations are now organized into unique module called `mitiq.interface`.
+
+In the context of probabilistic error cancellation, the sub-module `mitiq.pec.representations` has been significantly improved. Now one can easily compute the optimal quasi-probabiliy representation of an ideal gate as a linear combination of `NoisyOperation` objects.
```

Comment:
I can't seem to find the `mitiq.pec.representations` submodule in the codebase, maybe I am just missing something, or maybe it is named something else?

Reply:
@andreamari has this been renamed to https://github.com/unitaryfund/mitiq/blob/master/mitiq/pec/sampling.py?

Reply:
I ment this folder: `https://github.com/unitaryfund/mitiq/tree/master/mitiq/pec/representations`
It also has an `__init__.py` file so I think it can be considered as a sub-module, right?